test/karma-test/test.js

```diff
@@ -0,0 +1,35 @@
+var expect  = chai.expect;
+describe('Home Controllers', function(){
+	beforeEach(module('EPA_RFI'));
+	describe('homeCtrl',function(){
+		
+	});
+	var scope, ctrl, $httpBackend,httpHelper;
+	beforeEach(inject(function(_$httpBackend_,_httpHelper_, $rootScope, $controller) {
+		$httpBackend = _$httpBackend_;
+		httpHelper   = _httpHelper_;
+		scope ={};
+		ctrl = $controller('homeCtrl', {$scope: scope});
+	}));
+	it('should clear error boxes',inject(function($controller){
+		var homeController = ctrl
+		scope.clearErrorBoxes();
+		expect(scope.noResponseError).to.be.false;
+		expect(scope.invalidInputError).to.be.false;
+	}));
+	it('service should get uvindexes',function(){
+		var testResut = {"success":true,"data":[
+			{"SEQUENCE":1,"CITY":"ATLANTA","STATE":"GA",
+			"DATE_TIME":"DEC/29/2015 06 AM","UV_VALUE":0,"EXPOSURE":"Low"}]};
+		 $httpBackend.whenPOST('/api/getUVIndex')
+  			.respond(testResut);
+		 httpHelper.queryByLocation("ATLANTA","GA").then(function(retval) {
+			expect(retval.data.success).to.be.true;
+			expect(retval.data.data["SEQUENCE"]).to.equal(testResut["data"]["SEQUENCE"]);
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>['SEQUENCE'] is better written in dot notation.<br>['data'] is better written in dot notation.